crates/sui-core/src/authority_aggregator.rs

```diff
@@ -1559,11 +1607,32 @@ where
                     non_retryable_errors: group_errors(state.non_retryable_errors),
                 }
             }
-        })
+        })?;
+
+        // Use best efforts to send the cert to remaining validators.
+        let metrics = self.metrics.clone();
+        spawn_monitored_task!(async move {
+            let mut timeout = Box::pin(sleep(timeout_after_quorum));
+            loop {
+                tokio::select! {
+                    _ = &mut timeout => {
+                        debug!(?tx_digest, "Timed out in post quorum cert broadcasting: {:?}", timeout_after_quorum);
+                        metrics.cert_broadcasting_post_quorum_timeout.inc();
+                        break;
+                    }
+                    res = remaining_tasks.next() => {
+                        if res.is_none() {
```

Comment:
Do we have a metric anywhere that shows how many remaining_tasks we do have? I am curious to see how frequently we hit the case where we have `remaining_tasks` - which could confirm part of the theory here

Reply:
adde in the latest commit